modules/skeletonViewer/skeletonViewer.xml

```diff
@@ -26,7 +26,7 @@
     <param default="off" desc="Switch it on to display the floor.">show-floor</param>
     <param default="(0.0 0.0 0.0)" desc="The floor center in world coordinates.">floor-center</param>
     <param default="(0.0 0.0 1.0)" desc="The floor normal in world coordinates.">floor-normal</param>
-    <param default="(10 10)" desc="Grid resolution of the floor expressed in number of squares per side">grid-resolution</param>
+    <param default="(10 10)" desc="Grid resolution of the floor expressed in number of squares per side. Since the floor size 10x10 meters, a resolution of e.g. (100 100) will create a grid of squares with side length 10cm.">grid-resolution</param>
```

Comment:
```suggestion
    <param default="(10 10)" desc="Grid resolution of the floor expressed in number of squares per side. Since the floor size is 10x10 square meters, a resolution of e.g. (100 100) will create a grid of squares with side length 10 cm.">grid-resolution</param>
```